From 56192abb8ee316c132f256f20fcd0ef47c1a69ba Mon Sep 17 00:00:00 2001
From: Hec <hec@heccraft.com>
Date: Thu, 25 Jun 2026 02:50:09 -0400
Subject: [PATCH 19/19] FDSDK 26.08: Use git_repo for
 core-deps/python-dbusmock.bst

When pulling this particular project from pypi, the build gives the
error "LookupError: setuptools-scm was unable to detect version for
/buildstream/gnome/core-deps/python-dbusmock.bst."

Changing to using the git repo fixes this entirely, it seems. I can
trace this error back to this[1] commit in fdSDK.

[1]: https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/commit/342c3c1500388a817c6f283971710b543b1dd27b
---
 elements/core-deps/python-dbusmock.bst | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/elements/core-deps/python-dbusmock.bst b/elements/core-deps/python-dbusmock.bst
index 4b715e393..a606e9f23 100644
--- a/elements/core-deps/python-dbusmock.bst
+++ b/elements/core-deps/python-dbusmock.bst
@@ -1,10 +1,10 @@
 kind: pyproject
 
 sources:
-- kind: tar
-  url: 
-    pypi:a1/58/a3994e328b33e4cfa6ea9b6213c1ddff2fffdd6b342560fe92a400ab9eaf/python_dbusmock-0.36.0.tar.gz
-  ref: 2d3812ee6c1e15607bca882ed5dfcabaac449c1a3b9627080bbab3deefd56fd2
+- kind: git_repo
+  url: github:martinpitt/python-dbusmock.git
+  track: '*'
+  ref: 0.38.1-0-gee00606c4a57d83028ae2df71d47f6bda07741f8
 
 build-depends:
 - freedesktop-sdk.bst:public-stacks/buildsystem-python-setuptools.bst
-- 
2.54.0

